extensions/polished/src/convert-hex.tsx

```diff
@@ -0,0 +1,45 @@
+import { ActionPanel, Form, SubmitFormAction, useNavigation, showToast, ToastStyle } from "@raycast/api"
+import { rgba, parseToHsl } from "polished"
+import { Results } from "./components"
+
+type Values = {
+  hex: string
+  alpha: string
+}
+
+export default function ConvertHex() {
+  const { push } = useNavigation()
+
+  function handleSubmit(values: Values): void {
+    const { hex, alpha = "1" } = values
+    const a = parseFloat(alpha)
```

Comment:
Looks like the valid range for the alpha component should be validated here. When testing the extension locally here, I run into an assertion when converting colors (I guess at the point where the color would be shown in the list...).

Reply:
Please also check the other commands where an invalid alpha value could be entered.